pkg/merkle/tree.go

```diff
@@ -54,146 +52,67 @@ type Proof struct {
 var EmptyLeafValue = new(uint256.Int).Mod(
 	uint256.MustFromBig(new(big.Int).SetBytes(makeSeedForEmptyLeaf())),
 	uint256.MustFromBig(ff.Modulus()),
-)
+).String()
```

Comment:
I don't agree to call `.String()` here. Value should be a type of value (which is `uint256` in our case). String is the way we happen to serialize it